bugtool/cmd/configuration.go

```diff
@@ -249,7 +251,7 @@ func k8sCommands(allCommands []string, pods []string) []string {
 	var commands = []string{
 		"kubectl get nodes -o wide",
 		"kubectl describe nodes",
-		"kubectl get pods,svc --all-namespaces",
+		"kubectl get pods,svc,cnp,netpol --all-namespaces -o wide",
```

Comment:
This tool is meant to be used by people who want to send us information about their setups; I doubt that they will want us to see their network / security policies by default, as this could be a security risk. We can add a bugtool configuration file to run whatever commands we want during the CI:

```
vagrant@runtime:~/go/src/github.com/cilium/cilium/test$ cilium-bugtool --help | grep "\-\-config"
      --config string           Configuration to decide what should be run (default "./.cilium-bugtool.config")
```

example of such a file:
```
{
	"commands": [
		"ps",
		"hostname",
		"ip a",
		"ip r",
		"ip link",
		"uname -a",
		"dig",
		"netstat",
		"pidstat",
		"arp",
		"top -b -n 1",
		"uptime",
		"dmesg",
		"bpftool map show",
		"bpftool prog show",
		"docker version",
		"journalctl -u cilium*",
		"journalctl -u kubelet",
		"iptables-save",
		"iptables -S",
		"ip6tables -S",
		"iptables -L -v",
		"cat /proc/sys/net/core/bpf_jit_enable",
		"cat /proc/kallsyms",
		"cat /etc/resolv.conf",
		"ethtool lxc0c2de",
		"ethtool -i lxc0c2de",
		"ethtool lo",
		"ethtool -i lo",
		"ethtool lxcff273",
		"ethtool -i lxcff273",
		"ethtool enp0s3",
		"ethtool -i enp0s3",
		"ethtool docker0",
		"ethtool -i docker0",
		"ethtool lxc8b727",
		"ethtool -i lxc8b727",
		"ethtool veth1be11f9",
		"ethtool -i veth1be11f9",
		"ethtool lxcb70a9",
		"ethtool -i lxcb70a9",
		"ethtool vethf57e3f2",
		"ethtool -i vethf57e3f2",
		"ethtool cilium_net",
		"ethtool -i cilium_net",
		"ethtool vethc53dbee",
		"ethtool -i vethc53dbee",
		"ethtool cilium_host",
		"ethtool -i cilium_host",
		"ethtool cilium_health",
		"ethtool -i cilium_health",
		"ethtool cilium_vxlan",
		"ethtool -i cilium_vxlan",
		"ethtool lxc57276",
		"ethtool -i lxc57276",
		"ethtool lxc3ebac",
		"ethtool -i lxc3ebac",
		"cp /boot/config-4.9.13-040913-generic /tmp/cilium-bugtool-20180303-224501.949+0000-UTC-101436355/conf/kernel-config-4.9.13-040913-generic",
		"cp -r /var/run/cilium/state /tmp/cilium-bugtool-20180303-224501.949+0000-UTC-101436355/cmd/state",
		"cilium debuginfo",
		"cilium config",
		"cilium bpf tunnel list",
		"cilium bpf lb list",
		"cilium bpf endpoint list",
		"cilium bpf ct list global",
		"cilium status --verbose",
		"kubectl describe nodes",
		"kubectl get pods,svc --all-namespaces",
		"kubectl version"
	]
}
```